clients/cli/cmd/internal/push.go

```diff
@@ -202,6 +209,9 @@ func (source *Source) Push(client *phrase.APIClient, waitForResults bool, branch
 			fmt.Fprintln(os.Stderr, strings.Repeat("-", 10))
 		}
 	}
+	if !noErrors {
+		return errors.New("not_all_files_processed")
```

Comment:
Maybe instead of an error code like looking message we could write a short sentence what happened?

Reply:
:heavy_check_mark: following other cases in the codebase, I rewrote this as a (semi)sentence